### ISSUE TITLE
Expand links get squished when line numbers are hidden, in unified (non-split) mode

### ISSUE THREAD

**Issue by MarkSiedle** [praneshr/react-diff-viewer#64]
Firstly thanks for this great library, amazing work :)

I believe there's a minor UI issue when using unified (non-split) view and hiding line numbers, where the expand links end up squished over to the right, like this:

![2020-01-09 13_15_53-Audit - Octopus Deploy](https://user-images.githubusercontent.com/819605/72034957-3a47c080-32e2-11ea-9b72-4b4b8ce941f9.png)

The combination to get this to occur is as follows:
```
return <ReactDiffViewer ... splitView={false} hideLineNumbers={true} />;
```

**Comment by praneshr** [comment]
Hi @MarkSiedle, thanks for reporting this bug. I'll take a look.

**Comment by praneshr** [comment]
Will be fixed when https://github.com/praneshr/react-diff-viewer/pull/61 is merged.

**Comment by praneshr** [comment]
Fixed in [v3.1.0](https://github.com/praneshr/react-diff-viewer/releases/tag/v3.1.0)
